sdk/basyx/object_store.py

```diff
@@ -10,12 +10,12 @@
 """
 
 import abc
+import typing
 from typing import MutableSet, Iterator, Generic, TypeVar, Dict, List, Optional, Iterable
```

Comment:
Either, I'd import typing and always reference all objects via `typing.MutableSet` etc. or I'd only import the specific objects we need. Mixing these two styles in one module seems confusing to me.